templates/ingress.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: extensions/v1beta1
```

Comment:
the whole ingress block should be in within an `if`.

if ingress isn't enabled in values.yaml, this shouldn't be rendered